syntax = "proto2";

package msg.proto;

enum CommandType {
    COMMAND_LS = 0;
    COMMAND_DOWNFILE = 1;
    COMMAND_ERRORMSG = 2;
};

message CommandMsg{
    required CommandType command = 1;
    optional bytes args = 2;
    optional bytes msg = 3;
}
